compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -253,6 +256,8 @@ static void build_getter(AggregateType* ct, Symbol *field) {
 
   if (fieldIsConst)
     fn->addFlag(FLAG_REF_TO_CONST);
+  else if (recordLike)
```

Comment:
It doesn't make it less verbose, but I finally figured out this means
  FLAG_REF_TO_CONST_INHERITS_THIS_CONST
(to me, at least).  The reference is const if 'this' is const.